src/main/java/com/kuzmins/controllers/TicketController.java

```diff
@@ -0,0 +1,26 @@
+package com.kuzmins.controllers;
+
+
+import com.kuzmins.config.ConditionalBean;
+import com.kuzmins.model.tickets.Ticket;
+import com.kuzmins.service.TicketService;
+import lombok.AllArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@AllArgsConstructor
+public class TicketController {
+
+    private final TicketService ticketService;
+    private final ConditionalBean conditionalBean;
+
+    @GetMapping("/ticket/{id}")
```

Comment:
teh resources in url have to in plural form `tickets`

Reply:
fixed